### PR TITLE
fix/use-server-time

### LINKED ISSUE #2559: Refer last 30 minutes on the current time of the packetfence server
Searching for the last 30 minutes entries use the browser time and not the packetfence's server time.

### PR DESCRIPTION
**Pull request by jrouzierinverse**
Description
===========

Impacts
=======
Admin

Issue
=====
fixes #2559

Delete branch after merge
=========================
NO

NEWS file entries
=================
Bug Fixes
------------

* Using the system when instead of browser time when using preset time values (#2559)

UPGRADE file entries
====================
(OPTIONAL, but may be required...)

### CODE REVIEW

**Comment by dwlfrth** [comment]
Is this ready to be merged ?
If not, can we make it ready (and merge) before end of day.

**Comment by jrouzierinverse** [comment]
Yes it is ready who is going to review it?

File: html/pfappserver/lib/pfappserver/PacketFence/Controller/Admin.pm [hunk 1]
```diff
@@ -350,6 +352,34 @@ sub configuration :Chained('object') :PathPart('configuration') :Args(0) {
 
 }
 
+=head2 time_offset
+
+time_offset
```
Comment (dwlfrth) [inline html/pfappserver/lib/pfappserver/PacketFence/Controller/Admin.pm]: That is a name, not a description

**Review by dwlfrth — CHANGES_REQUESTED** [review]

File: html/pfappserver/lib/pfappserver/PacketFence/Controller/Admin.pm (hunk 1 above)
Comment (louismunro) [reply html/pfappserver/lib/pfappserver/PacketFence/Controller/Admin.pm]: Agreed, we should have at least one sentence to describe what the method is supposed to do and what the params are.

**Comment by jrouzierinverse** [comment]
Done

**Comment by louismunro** [comment]
Looks good then.

**Pull request merged**